paddle/fluid/operators/CMakeLists.txt

```diff
@@ -254,6 +254,9 @@ op_library(cross_entropy_op DEPS cross_entropy)
 if(WITH_GPU)
   op_library(softmax_with_cross_entropy_op DEPS cross_entropy softmax cub)
   op_library(sequence_softmax_op DEPS cub)
+elseif(WITH_AMD_GPU)
+  op_library(softmax_with_cross_entropy_op DEPS cross_entropy softmax rocprim)
+  op_library(sequence_softmax_op DEPS rocprim)
```

Comment:
The header file is not included in the operators ".cc", so why need to depend on `rocprim` here?

Reply:
That's a WA towards HIP/HCC toolchain, some symbol will be missing without "whole-archive".

Reply:
Instead of including cub, HIP/HCC build will refer to the header files of rocprim.